.circleci/run_acceptance.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+export BROWSERSTACK_USE_AUTOMATE="1"
+export BROWSERSTACK_PROJECT_NAME="Answers SDK"
+export BROWSERSTACK_BUILD_ID=$CIRCLE_BRANCH
+COMMIT_MSG_TITLE=$(git log -n 1 --pretty=format:%s)
```

Comment:
Should we set the `BROWSERSTACK_BUILD_ID` to something that is unique? I would think there would be collisions if we use `CIRCLE_BRANCH`. Would `CIRCLE_BUILD_NUM` be a better choice here because those numbers are unique for each job?

Reply:
I was trying to group them by branch name, so the automate ui would look like
![image](https://user-images.githubusercontent.com/23005393/95383393-1b9a4880-08b9-11eb-8555-07a5dcfcda9d.png)
but I guess if somebody were to use the same branch name for two different prs that could be annoying

Reply:
It looks like the `BROWSERSTACK_BUILD_ID` doesn't mind having multiple entries of the same value, so I think works.

Reply:
I still think what you said makes sense though, updated it to use CIRCLE_BUILD_NUM.

Reply:
Alternatively, we could append `CIRCLE_BUILD_NUM` to `CIRCLE_BRANCH`